aries/aries_vcx_core/src/anoncreds/anoncreds/mod.rs

```diff
@@ -1079,17 +1096,17 @@ impl BaseAnonCreds for Anoncreds {
         wallet: &impl BaseWallet,
         prover_did: &Did,
         cred_offer_json: &str,
-        cred_def_json: &str,
+        cred_def_json: CredentialDefinition,
         master_secret_id: &str,
     ) -> VcxCoreResult<(String, String)> {
-        let mut cred_def_json: Value = serde_json::from_str(cred_def_json)?;
-        let cred_def_id = cred_def_json["id"].as_str().unwrap().to_string();
-        cred_def_json.as_object_mut().unwrap().insert(
-            "issuerId".to_owned(),
-            cred_def_id.split(':').next().unwrap().into(),
-        );
-        let cred_def: CredentialDefinition = serde_json::from_str(&cred_def_json.to_string())?;
-        let credential_offer: CredentialOffer = serde_json::from_str(cred_offer_json)?;
+        // let mut cred_def_json: Value = serde_json::from_str(cred_def_json)?;
+        // let cred_def_id = cred_def_json["id"].as_str().unwrap().to_string();
+        // cred_def_json.as_object_mut().unwrap().insert(
+        //     "issuerId".to_owned(),
+        //     cred_def_id.split(':').next().unwrap().into(),
+        // );
```

Comment:
can be deleted?

Reply:
Thank you for catching this, removed in the next PR